Authorization/Authorization/Presentation/Login/SignInViewModel.swift

```diff
@@ -19,6 +19,8 @@ public class SignInViewModel: ObservableObject {
     @Published private(set) var isShowProgress = false
     @Published private(set) var showError: Bool = false
     @Published private(set) var showAlert: Bool = false
+    public var sourceScreen: LogistrationSourceScreen
```

Comment:
The public modifier here is redundant.
var can be replaced with let.
`public var sourceScreen: LogistrationSourceScreen` -> `let sourceScreen: LogistrationSourceScreen`